docs/import/color_function_util.js

```diff
@@ -21,62 +21,70 @@ let globalTd;
  * DOM elements with attached on-change handlers.
  */
 function populateColorFunctions() {
-  const colorFunctionDiv = $('#color-fxn-editor');
-  colorFunctionDiv.prepend(createRadioFor(ColorStyle.SINGLE));
-  colorFunctionDiv.prepend(createRadioFor(ColorStyle.DISCRETE));
-  colorFunctionDiv.prepend(createRadioFor(ColorStyle.CONTINUOUS));
+  colorStyleTypeStrings.forEach(
+      (typeAsString, colorStyle) =>
+          $('#' + typeAsString + '-radio')
+              .on('change', () => switchSchema(colorStyle)));
+  $('#property-radio').on('change', () => {
+    const lastByPropertyType =
+        colorStyleTypeStrings.get(getColorFunction().lastByPropertyStyle);
+    $('#' + lastByPropertyType + '-radio')
+        .prop('checked', true)
+        .trigger('change');
+  });
 
   const singleColorPicker = createColorPicker('single-color-picker');
   singleColorPicker.on('change', () => setColor(singleColorPicker));
   $('#single').append(
-      createLabelFor(singleColorPicker, 'color'), singleColorPicker);
+      createLabelForMandatoryPicker(singleColorPicker, 'color'),
+      singleColorPicker);
 
   const continuousColorPicker = createColorPicker('continuous-color-picker');
   continuousColorPicker.on('change', () => setColor(continuousColorPicker));
-  const continuousPropertyPicker =
-      $(document.createElement('select'))
-          .prop('id', 'continuous-property-picker');
-  continuousPropertyPicker.on('change', () => {
-    setProperty(continuousPropertyPicker);
-    maybeDisplayMinMax(continuousPropertyPicker);
+
+  // TODO: make an educated guess about if this property should
+  // be continuous or discrete (based on # distinct vals?)
+  const propertyPicker = $('#property-picker').on('change', () => {
+    const property = propertyPicker.val();
+    setProperty(property);
+    switch (getColorFunction().currentStyle) {
+      case ColorStyle.CONTINUOUS:
+        maybeDisplayMinMax(property);
+        break;
+      case ColorStyle.DISCRETE:
+        populateDiscreteColorPickers(property);
+        break;
+      case ColorStyle.SINGLE:
+        throw Error('Somehow tried to set property while in single color mode');
```

Comment:
nit: will this get caught anywhere? Consider showing a snackbar error instead/in addition.

Reply:
Now with more snacks!